active/0001-automated-releases.md

```diff
@@ -0,0 +1,93 @@
+- Automated Releases
+- @kleimkuhler, @alpeb
+- 2020-02-17
+- 2020-03-03
+- RFC PR: [linkerd/rfc#7](https://github.com/linkerd/rfc/pull/7)
+- Linkerd Issue: [linkerd/linkerd2#3021](https://github.com/linkerd/linkerd2/issues/3021)
+- Reviewers: @grampelberg
+
+# Summary
+
+[summary]: #summary
+
+The goal for this project is to make the Linkerd2 release process as automated
+as possible. The process currently requires that the responsible member take
+multiple manual steps in order for the release artifacts to be properly built,
+tested, and updated for installations.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+The manual steps required in order to perform a release can lead to any number
+of errors by fault of the responsible member. These include errors such as: 
+- forgetting to perform a particular step
+- completing the steps in an incorrect order
+- dealing with flakiness related to the timing of when steps are completed
+
+Additionally, the member must "babysit" the individual steps in order to start
+the next one. This can be time consuming.
+
+Instead of addressing these points individually, the release process should be
+re-evaluated as a whole in order to utilize the CI workflow to automate the
+process where possible.
+
+Once the release process is changed, the responsible member will still be
+required to summarize the change notes and receive feedback by opening a pull
+request. Once that pull request is merged, the member will create a release
+tag which triggers the release process. From there, the building, testing, and
+uploading of the release artifacts will be automated. Once those steps
+complete, the member will send out announcement emails to the CNCF mailing
+lists.
+
```

Comment:
I'd love to see a list of the steps that we're planning on automating and which ones should stay manual. Especially some requirements here.

Reply:
This feels kind of like the discussion happening in #8, but I had that as part of the implementation details. I tried to keep this as a high-level as I could.

Reply:
There are high-level steps now. Let me know if that helps!